environment.yml

```diff
@@ -60,17 +64,7 @@ dependencies:
 
   # downstream packages
   - aiobotocore<2.0.0  # GH#44311 pinned to fix docbuild
-  - botocore
-  - cftime
-  - dask
-  - ipython
-  - seaborn
```

Comment:
nvm, looks like we still need seaborn for now

I don't think it's a heavy dependency, let's just keep it in for now?

Reply:
Yeah, it's still used in the docs but just for a  colormap, so need to update that section to use a matplotlib built-in colormap.

Reply:
shall we do that separately and remove the rest of the (much heavier) dependencies? seaborn isn't that big

Reply:
I think I will just do that here, it's only a small change, and seaborn does bring in statsmodels

Reply:
no objections if it's a small change, though don't worry about it if it turns out to be more effort, looks like statsmodels is optional for seaborn (don't know if it was required in the past?)
```
(.venv) marcogorelli@DESKTOP-U8OKFP3:~/tmp$ pip install seaborn
Collecting seaborn
  Using cached seaborn-0.12.1-py3-none-any.whl (288 kB)
Requirement already satisfied: pandas>=0.25 in ./.venv/lib/python3.8/site-packages (from seaborn) (1.4.4)
Requirement already satisfied: numpy>=1.17 in ./.venv/lib/python3.8/site-packages (from seaborn) (1.25.0.dev0+165.g6f9237e91)
Collecting matplotlib!=3.6.1,>=3.1
  Using cached matplotlib-3.6.2-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (9.4 MB)
Collecting pillow>=6.2.0
  Using cached Pillow-9.3.0-cp38-cp38-manylinux_2_28_x86_64.whl (3.3 MB)
Requirement already satisfied: python-dateutil>=2.7 in ./.venv/lib/python3.8/site-packages (from matplotlib!=3.6.1,>=3.1->seaborn) (2.8.2)
Collecting cycler>=0.10
  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
Requirement already satisfied: packaging>=20.0 in ./.venv/lib/python3.8/site-packages (from matplotlib!=3.6.1,>=3.1->seaborn) (21.3)
Requirement already satisfied: pyparsing>=2.2.1 in ./.venv/lib/python3.8/site-packages (from matplotlib!=3.6.1,>=3.1->seaborn) (3.0.9)
Collecting fonttools>=4.22.0
  Using cached fonttools-4.38.0-py3-none-any.whl (965 kB)
Collecting kiwisolver>=1.0.1
  Using cached kiwisolver-1.4.4-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.2 MB)
Collecting contourpy>=1.0.1
  Using cached contourpy-1.0.6-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (295 kB)
Requirement already satisfied: pytz>=2020.1 in ./.venv/lib/python3.8/site-packages (from pandas>=0.25->seaborn) (2022.6)
Requirement already satisfied: six>=1.5 in ./.venv/lib/python3.8/site-packages (from python-dateutil>=2.7->matplotlib!=3.6.1,>=3.1->seaborn) (1.16.0)
Installing collected packages: pillow, kiwisolver, fonttools, cycler, contourpy, matplotlib, seaborn
Successfully installed contourpy-1.0.6 cycler-0.11.0 fonttools-4.38.0 kiwisolver-1.4.4 matplotlib-3.6.2 pillow-9.3.0 seaborn-0.12.1
(.venv) marcogorelli@DESKTOP-U8OKFP3:~/tmp$ . repl.sh

In [1]: import statsmodels
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In [1], line 1
----> 1 import statsmodels

ModuleNotFoundError: No module named 'statsmodels'

```